src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -298,6 +299,9 @@ def load_arguments(self, _):
                    help='Provide a string value of a Storage Account in the provided Resource Group. Or Resource ID of a Storage Account in a different Resource Group')
         c.argument('consumption_plan_location', options_list=['--consumption-plan-location', '-c'],
                    help="Geographic location where Function App will be hosted. Use 'functionapp list-consumption-locations' to view available locations.")
+        c.argument('runtime', help='The function runtime stack. Currently supported for Linux apps only', arg_type=get_enum_type(['dotnet', 'node', 'python']))
```

Comment:
I have a question. How come os_types is defined in _params.py, but runtimes are defined in the call to get_enum_type?

Reply:
I am expecting the OS_type to be re-used across other commands in the future so i made this a global & then used as arg_type=get_enum_type(OS_TYPES) in this specific command. The runtimes, should be specific to this command so i left the enum values in the command itself. Hope this makes sense.

Reply:
Ah I see, that makes sense! Thanks for explaining.